### PR TITLE
Flow end flags/v1

### PR DESCRIPTION
**Pull request by victorjulien**
Small cleanup as well as making some "flag" space.

### CODE REVIEW

**Comment by suricata-qa** [comment]
WARNING:
 
 | field | baseline | test | % |
|--|--|--|--| 
| | SURI_TLPR1_stats_chk | | |
| .uptime | 649 | 624 | 96.15% | 

Pipeline 22685

File: src/flow.h [hunk 1]
```diff
@@ -240,14 +240,10 @@ typedef struct AppLayerParserState_ AppLayerParserState;
...
-#define FLOW_END_FLAG_STATE_NEW         0x01
-#define FLOW_END_FLAG_STATE_ESTABLISHED 0x02
-#define FLOW_END_FLAG_STATE_CLOSED      0x04
-#define FLOW_END_FLAG_EMERGENCY         0x08
-#define FLOW_END_FLAG_TIMEOUT           0x10
-#define FLOW_END_FLAG_FORCED            0x20
-#define FLOW_END_FLAG_SHUTDOWN          0x40
-#define FLOW_END_FLAG_STATE_BYPASSED    0x80
+#define FLOW_END_FLAG_EMERGENCY 0x01
+#define FLOW_END_FLAG_TIMEOUT   0x03
```
Comment (lukashino) [inline src/flow.h:244]: why is this 3,4,8 and not 2,3,4?

Comment (lukashino) [reply src/flow.h:244]: or 2, 4, 8

Comment (victorjulien) [reply src/flow.h:244]: typo, good catch

Comment (victorjulien) [reply src/flow.h:244]: #11796

**Comment by victorjulien** [comment]
replaced by #11796

**Pull request closed without merging**